prow/plugins/assign/assign.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package assign
+
+import (
+	"regexp"
+	"strings"
+
+	"github.com/Sirupsen/logrus"
+
+	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/prow/plugins"
+)
+
+const pluginName = "assign"
+
+var acceptRe = regexp.MustCompile(`(?mi)^\/(un)?assign(( [-\w]+?)*)\r?$`)
```

Comment:
Ack

Generally I think flexibility tends to be the enemy of comprehensibility and predictability. So I try to avoid flexibility unless it is needed.

I added support for trailing whitespace since github adds this automatically. Otherwise I will be surprised if lots of people start doing things like `/assign         @apelisse                     @fejta` but we can add that if we start getting complaints. Until then I'd prefer to maximize the legibility of the regex string. "foo bar" is easier on the eyes than "\s+foo\s+bar\s+"

Is there a reason the group needs to be non-capturable? It also makes the regex harder to read.